docs/users-manual/submitting-a-job.rst

```diff
@@ -124,8 +124,8 @@ we tell HTCondor which input file to send to each instance of the program.
     executable     = foo
     arguments      = input_file.$(Process)
 
-    request_memory = 4096
     request_cpus   = 1
+    request_memory = 4096
```

Comment:
Not sure if we want to add unit specifiers for this and the other example at line 1349. Otherwise LGTM